README.md

```diff
@@ -52,9 +52,9 @@ Modules
 -------
 * spock-core -- Core framework. This is the only mandatory module.
 * spock-specs -- Specifications for spock-core, implemented using Spock. Not required for using Spock.
-* spock-spring -- Integration with the [Spring TestContext Framework](http://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
-* spock-tapestry -- Integration with the [Tapestry 5](http://tapestry.apache.org/tapestry5/) IoC container.
-* spock-guice -- Integration with [Guice](http://code.google.com/p/google-guice/) 2/3.
+* spock-spring -- Integration with the [Spring TestContext Framework](https://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
+* spock-tapestry -- Integration with the [Tapestry 5](https://tapestry.apache.org/tapestry5/) IoC container.
```

Comment:
https://tapestry.apache.org/ would be enough

Reply:
https://github.com/spockframework/spock/pull/1133/commits/7d260955a8f2088945e5000818914e2bd1f6cb0d